src/Microsoft.Framework.ActivatorUtilities.Sources/project.json

```diff
@@ -1,15 +1,19 @@
 ﻿{
-  "version": "1.0.0-*",
-  "shared": "*.cs",
-  "dependencies": {
-  },
-  "frameworks" : {
-    "net45": { },
-    "dnx451": { },
-    "dnxcore50": {
-        "dependencies": {
-            "System.Runtime": "4.0.20-*"
-        }
+    "version": "1.0.0-*", 
+    "shared": "*.cs", 
+    "dependencies": { }, 
+    "frameworks" : {
+        "net45": { }, 
+        "dnx451": { }, 
+        "dnxcore50": {
+            "dependencies": {
+                "System.Runtime": "4.0.20-*"
+            } 
+        },
+        "dotnet": {
```

Comment:
We know this will work. We can only have dotnet when we validate that it works everywhere